virus/Obeliscus_oris.md

```diff
@@ -0,0 +1,322 @@
+# _Obeliscus oris_: An Uncharacterised Circular RNA Encoding a Single Protein, Oblin-1
+written by: [Isha Singh](https://github.com/its-isha-singh)
+
+### _Obeliscus oris_
+_Obeliscus oris_ translates to _obelisk of the mouth_. All samples containing the
+obelisk (excluding one) were isolated from human oral-nasopharyngeal passages,
+hence the name _Obeliscus oris_.
+
+![](img/obeliscus_oris/cover.jpeg)
```

Comment:
Please add "tittles" to the figures in the square brackets so text is displayed if hte image does not load.

Reply:
I've made the changes. Let me know if they went through!